plugins/game/lib/api/filters/KalturaUserScorePropertiesFilter.php

```diff
@@ -154,12 +154,17 @@ protected function getListAllUsers($redisWrapper, $redisKey)
 	protected function paginateResults($pager, $results)
 	{
 		$startRank = ($pager->pageIndex - 1) * $pager->pageSize;
```

Comment:
Min level 0 can be done in a one line - $startRank = min (($pager->pageIndex - 1) * $pager->pageSize,0)